Makefile

```diff
@@ -1074,6 +1086,13 @@ ggml/src/ggml-aarch64.o: \
 	ggml/src/ggml-common.h
 	$(CC) $(CFLAGS)    -c $< -o $@
 
+#$(CXX) $(CXXFLAGS) -std=c++17 -fopt-info-vec-missed -c $< -o $@
+ggml/src/ggml-fp8.o: \
+	ggml/src/ggml-fp8.cpp \
+	ggml/src/ggml-fp8.h \
+	ggml/src/ggml-common.h
+	$(CXX) $(CXXFLAGS) -std=c++17 -c $< -o $@
```

Comment:
Should target `-std=c++11`

Reply:
I use "static constexpr" in this file that is not allowed with c++11 (and may be more).
And because it is the default with gcc, I develop/test the FP8 template with it.
I see you have it use for GGML_SYCL build to.

So I can try to rewrite it with c++11 before merge request, but is it really needed? Do you have target that do not support it?

Reply:
Yes, this is not a good reason to change the project standard.

Reply:
You have it for GGML_SYCL build, is there a reason to keep that old standart that was the 1er draft for modern C++?

I'll have a try to rewrite it with only c++11 and see how it look.

Reply:
Backends are for the most part self-contained and don't affect the rest of the project. What you are trying to merge here would be part of the core ggml and would require changing the standard for the entire project. I would also prefer to use C++17, but that is a discussion for another time.